### PR TITLE
add new email file per bug1491135

### PR DESCRIPTION
**Pull request by peiying2**

### CODE REVIEW

**Comment by peiying2** [comment]
Not ready for review yet. Need to check against final mockups.

**Comment by peiying2** [comment]
@flodolo ready for review. 
- Mockups: https://drive.google.com/drive/folders/1rRo-WSeLjBhphR8Vgfs1vFw_xt7SOzDy
- Source: https://docs.google.com/spreadsheets/d/1jM7CMqAjH1fwZbMvxK0DoOyc_LBbLExY-TAa8TixDeU/edit#gid=947968131

File: en-US/emails/2018/firefox_account_welcome_journey.lang [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+Switch screens without a hitch. Open a link on one screen and view it instantly on any of your synced devices.
+
+
+# CTA
+;View Sync Settings
+View Sync Settings
+
+
+# Sign off
+;Questions, we're here to <a href=" https://support.mozilla.org" >help</a>.
```
Comment (flodolo) [inline en-US/emails/2018/firefox_account_welcome_journey.lang]: No spaces in the URL or the tags

File: en-US/emails/2018/firefox_account_welcome_journey.lang [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+End-to-End Encryption
+
+
+# Body Subhead #1
+;Keeps your synced data on lockdown. Even we can’t see what you share.
+Keeps your synced data on lockdown. Even we can’t see what you share.
+
+
+# Body #2
+;Tracking Protection
```
Comment (flodolo) [inline en-US/emails/2018/firefox_account_welcome_journey.lang]: Why are we promoting something that is not in Firefox anymore? Is marketing aware that we moved away from Tracking Protection, given this is going out in 2 months?

File: en-US/emails/2018/firefox_account_welcome_journey.lang [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+
+# Body #1A: users who have not synced devices
+;Save time by <a href=" https://accounts.firefox.com/settings">syncing your devices</a>.
+Save time by <a href=" https://accounts.firefox.com/settings">syncing your devices</a>.
+
+
+# Body #1B: users who have synced devices
+;Already synced? Make Firefox your  <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">default browser</a>.
+Already synced? Make Firefox your default browser.
```
Comment (flodolo) [inline en-US/emails/2018/firefox_account_welcome_journey.lang]: Strings don't match

**Comment by peiying2** [comment]
Waiting for either quick word fix or drop the Privacy sections all together.

File: en-US/emails/2018/firefox_account_welcome_journey.lang [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+
+
+# Body #1B: users who have synced devices
+;Already synced? Make Firefox your <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">default browser</a>.
+Already synced? Make Firefox your <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">default browser</a>.
+
+
+# Body 2
+;Already synced? Make Firefox your <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">default browser</a>.
+Already synced? Make Firefox your <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">default browser</a>.
```
Comment (flodolo) [inline en-US/emails/2018/firefox_account_welcome_journey.lang]: Still duplicate.

**Comment by flodolo** [comment]
I rebased and pushed fixes in https://github.com/mozilla-l10n/engagement-l10n/pull/179/commits/29888540daee5a4e8a253ececce6640c7709efef, please make sure to not overwrite them.

**Pull request merged**